packages/js/src/plugins/auctionHouseModule/errors.ts

```diff
@@ -127,3 +127,17 @@ export class CreateListingRequiresSignerError extends AuctionHouseError {
     });
   }
 }
+
+/** @group Errors */
+export class WithdrawRequiresSignerError extends AuctionHouseError {
```

Comment:
Since this error is just for withdrawing from the buyer's account its name should be more explicit.

```suggestion
export class WithdrawFromBuyerAccountRequiresSignerError extends AuctionHouseError {
```